rom/laghos.cpp

```diff
@@ -980,7 +997,8 @@ int main(int argc, char *argv[])
                 outfile_offlineParam << romOptions.blast_energyFactor << " ";
                 outfile_offlineParam << dim << " ";
                 outfile_offlineParam << dt << " ";
-                outfile_offlineParam << source << endl;
+                outfile_offlineParam << source << " ";
+                outfile_offlineParam << romOptions.atwoodFactor << " ";
```

Comment:
In the previous version the file ended with `endl`, but now with a space. I guess it doesn't matter, but I also don't see a reason to have the space at the end.

Reply:
Thanks Dylan for bringing this to attention. This is related to a change that I plan to do in this PR but I forgot to finish it so it appears like as it is. I think `dim`, `dt` and `source` should probably go to another line (maybe the first line as it is about the problem setting rather than the ROM settings) of the parameter file. I will do the changes it tomorrow.